src/main/app/Resources/styles/print.less

```diff
@@ -0,0 +1,100 @@
+@media print {
+    html {
+        height: auto !important;
+        overflow-y: auto;
+    }
+
+    .app-header-container {
+        display: none;
+    }
+
+    .breadcrumb {
+        display: none;
+    }
+
+    .page-actions {
+        display: none;
+    }
+
+    .tag-cell {
+        display: flex;
+        flex-flow: row wrap;
+    }
+
+    .app .app-menu {
+        display: none;
+    }
+
+    .app-content .app-footer-container {
+        display: none;
+    }
+
+    .overview-user-actions {
+        display: none;
+    }
+
+    .data-list .list-header {
+        display: none;
+    }
+
+    // subject to change, not sure how to select by class:
+    #ajouter-une-session-de-formati {
```

Comment:
You cannot target ids because they are auto generated based on the label which is localized.